packages/zosfiles/src/cli/list/List.options.ts

```diff
@@ -51,5 +51,27 @@ export const ListOptions: {[key: string]: ICommandOptionDefinition} = {
         aliases: ["max"],
         description: strings.MAXLENGTH,
         type: "number"
+    },
+
+    /**
+     * The indicator that where ti file syste is mounted.
+     * @type {ICommandOptionDefinition}
+     */
+    path: {
```

Comment:
To make the parameters mutually exclusive you can use the `conflictsWith` property. See [ICommandOptionsDefinition.ts](https://github.com/zowe/imperative/blob/master/packages/cmd/src/doc/option/ICommandOptionDefinition.ts) for more details.